tools/packaging/static-build/firecracker/build-static-firecracker.sh

```diff
@@ -31,10 +31,11 @@ fi
 
 info "Build ${firecracker_repo} version: ${firecracker_version}"
 
-git clone ${firecracker_repo}
+[ -d firecracker ] || git clone ${firecracker_repo}
```

Comment:
you could use a variable for `firecracker` and use it here and in the line below.